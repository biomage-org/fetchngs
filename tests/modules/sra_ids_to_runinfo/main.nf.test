nextflow_process {

    name "Test Process SRA_IDS_TO_RUNINFO"
    script "modules/local/sra_ids_to_runinfo.nf"
    process "SRA_IDS_TO_RUNINFO"

    test("Should run without failures") {

        when {
            process {
                """
                input[0] = 'SRR9984183'
                input[1] = ''
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
