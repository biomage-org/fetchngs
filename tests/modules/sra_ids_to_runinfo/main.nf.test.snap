{
    "Should run without failures": {
        "content": [
            {
                "versions": [
                    "versions.yml:md5,cd1056ca246a1ab3486fb6800f3012fb"
                ],
                "tsv": [
                    "SRR9984183.runinfo.tsv:md5,a69083928f4b856d428a681c291b7ad6"
                ]
            }
        ],
        "timestamp": "2022-11-11T04:20:20+0000"
    }
}